### PR TITLE
Remove quiet flag, so that it works in podman-remote

### PR DESCRIPTION
**Pull request by rhatdan**
Currently podman tests has

skip_if_remote "FIXME FIXME FIXME: 'Ignoring' warning is never seen" \
               "bud with preprocessor error"

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

#### What type of PR is this?

> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test 
> /kind feature
> /kind flake
> /kind other

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by rhatdan** [comment]
@edsantiago @flouthoc PTAL

**Comment by rhatdan** [comment]
@containers/podman-maintainers PTAL

**Comment by edsantiago** [comment]
Works as expected, podman and podman-remote, in my local treadmill checkout. Kind of embarrassing that I never noticed how easy this one was to fix. In my defense, I'm a lazy slacker.

/lgtm
/hold

**Review by flouthoc — APPROVED** [review]
LGTM

**Pull request merged**
